test/distributed/test_c10d.py

```diff
@@ -4218,100 +4218,121 @@ def forward(self, inp):
             x = checkpoint(self.l2, x)
             return x
 
-    def _test_ddp_checkpointing(self, checkpoint_once, process_group, use_bucket_view, find_unused_parameters=False):
+    def _prepare_dummy_data(self):
```

Comment:
Can you also enable all the tests in https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/distributed/pipe_with_ddp_test.py and verify they work with `set_static_graph`? Can be done in a separate PR.

Reply:
sounds good, I will add verify them in follow up PR